### ISSUE TITLE
kml has visibility off for every point

### ISSUE THREAD

**Issue by CapnBry** [stronnag/bbl2kml#14]
I've dumped a blackbox log to a kmz file and it worked great. However, the non-default Folder has every one of its points visibility set to false. This means when I load it up in earth.google.com/web/ it can't easily be toggled on and off. With just the default options, Flight Mode is visible, and RSSI has the no-eye (not visible) icon. Clicking the visible icon, nothing appears. If `-rssi` is added, then the flight mode exhibits this behavior but RSSI works. If -efficiency is added, then it matches the visibility of rssi (all disabled if Flight Mode is the default view, and usable if RSSI is the default view).
![image](https://user-images.githubusercontent.com/677183/111876149-d2741780-8973-11eb-9573-d60455096d05.png)

Looking [at the code](https://github.com/stronnag/bbl2kml/blob/master/pkg/kmlgen/kmlbuilder.go#L410), it sets all the points visibility to the parent's visibility. Is there an issue that prevents it from being `kml.Visibility(defviz))` but then `addPoints(..., true)` so that all the points are visible=true, but the Folder has the visibility set as it is now? Google Earth Web slows to a crawl if you try to enable each individual point, but works great just toggling the folder visibility.

**Comment by stronnag** [comment]
So, there is a difference in behaviour in the way desktop GE works v. the web version. I'm sure there is a "lost in time" reason why it's the way it is; I'll have look at it.

**Comment by stronnag** [comment]
What a mess; the OGC KML standard mandates the behaviour requested; however Google declines to implement in the desktop application as [ "changing this would break existing code" ](https://stackoverflow.com/questions/42353749/how-to-uncheck-folders-in-kml-file). As most the users use the desktop  (vice the web) application; the web app being somewhat unusable on my rural broadband), I'm going to follow Google's lead on this. However, the next release (tomorrow I hope) will (a) offer an option to override this and (b) have a configuration file if you want to make a different behaviour your personal default).

```
$ flightlog2kml -help
Usage of flightlog2kml [options] file...
 ...
  -visibility int
    	0=folder value,-1=don't set,1=all on
```

Anything other than the default of 0 causes the desktop client to misbehave, and the web client to behave rationally.

I'll push the changes to `development` later today, prior to the later `master` merge and release.

**Comment by stronnag** [comment]
Test binaries 
[fl2x-0.9.7rc2.tar.gz](https://github.com/stronnag/bbl2kml/files/6176180/fl2x-0.9.7rc2.tar.gz)

**Comment by CapnBry** [comment]
Aha I figured there had to be a reason like the way it is supposed to be vs what works in various Google platforms. Nice that it behaves differently across the clients. I had been using the desktop Google Earth, but this machine didn't have it installed so I used the web platform for the first time.

The test binaries work, I tried all three options. `-visibility 0` gives me the regular behavior and both `-visibility 1` and `-visibility -1` allow me to toggle the entire track from the top level folder item in Google Earth web. Perfect! Also love the inclusion of the squares for RTH markers, that's a nice feature.
